base/tools/src/main/java/com/netscape/cmstools/CRMFPopClient.java

```diff
@@ -659,25 +659,12 @@ public KeyPair generateECCKeyPair(
             boolean temporary,
             int sensitive,
             int extractable) throws Exception {
-        /*
```

Comment:
I'm not sure I want to lose the info in the comments here.  How about add the following comment right above the generateECCKeyPair call below:
/*
 * ECDH_USAGES_MASK: used with SSL server cert that does ECDH ECDSA;
 *                                       can only be used with POP_NONE 
 * ECDHE_USAGES_MASK: used for other certs including SSL server cert that does ECDHE ECDSA
 * */

Reply:
OK, fixed.